### PR TITLE
SoundWire: log cleanups

### PR DESCRIPTION
**Pull request by plbossart**
cosmetic cleanups to remove warning for dapm widgets and make enumeration logs more consistent.

### CODE REVIEW

File: drivers/soundwire/slave.c [hunk 1]
```diff
@@ -163,13 +163,13 @@ int sdw_acpi_find_slaves(struct sdw_bus *bus)
 
 			if (id.unique_id != id2.unique_id) {
 				dev_dbg(bus->dev,
-					"Valid unique IDs %x %x for Slave mfg %x part %d\n",
+					"Valid unique IDs 0x%x 0x%x for Slave part_id 0x%04x, mfg_id 0x%04x\n",
```
Comment (bardliao) [inline drivers/soundwire/slave.c:166]: The order of part_id and mfg_id is wrong. It should ``mfg_id 0x%04x,  part_id 0x%04x``

File: drivers/soundwire/slave.c [hunk 2]
```diff
@@ -163,13 +163,13 @@ int sdw_acpi_find_slaves(struct sdw_bus *bus)
...
 					id.unique_id, id2.unique_id,
 					id.mfg_id, id.part_id);
 				ignore_unique_id = false;
 			} else {
 				dev_err(bus->dev,
-					"Invalid unique IDs %x %x for Slave mfg %x part %d\n",
+					"Invalid unique IDs 0x%x 0x%x for Slave part_id 0x%04x, mfg_id 0x%04x\n",
```
Comment (bardliao) [inline drivers/soundwire/slave.c:172]: ditto

**Review by bardliao — CHANGES_REQUESTED** [review]
The order of part_id and mfg_id is wrong. To speed up, I will go ahead to edit and create another PR for it since @plbossart is away this week.

**Comment by bardliao** [comment]
Replaced by https://github.com/thesofproject/linux/pull/2575

File: drivers/soundwire/slave.c (hunk 1 above)
Comment (plbossart) [reply drivers/soundwire/slave.c:166]: this was intentional, there are other logs where we write the part_id first.
If we put mfg_id first, we've got to change the other logs.

**Pull request closed without merging**